varsim.py

```diff
@@ -21,6 +23,51 @@
 REQUIRE_VARSIMJAR = not os.path.isfile(DEFAULT_VARSIMJAR)
 if REQUIRE_VARSIMJAR: DEFAULT_VARSIMJAR = None
 
+def convertCN(filenames, operation):
```

Comment:
This function does not seem to fix the VCF header accordingly to match the interpretation of CN